autonomy/cli/analyse.py

```diff
@@ -165,37 +168,47 @@ def parse_logs(file: Path) -> None:
         raise click.ClickException(str(e)) from e
 
 
-@abci_group.command(name="check-handlers")
-@click.argument(
-    "packages_dir",
-    type=click.Path(dir_okay=True, exists=True),
-    default=Path.cwd() / "packages",
-)
+@analyse_group.command(name="handlers")
+@pass_ctx
 @click.option(
-    "--skip",
+    "--common-handlers",
+    "-h",
     type=str,
-    default="abstract_abci",
-    help="Specify which skills to skip. Eg. skill_0,skill_1,skill_2",
+    default=[
+        "abci",
+    ],
+    help="Specify which handlers to check. Eg. -h handler_a -h handler_b -h handler_c",
+    multiple=True,
 )
 @click.option(
-    "--common",
+    "--ignore",
+    "-i",
     type=str,
-    default="abci",
-    help="Specify which handlers to check. Eg. handler_a,handler_b,handler_c",
+    default=[
+        "abstract_abci",
+    ],
+    help="Specify which skills to skip. Eg. -i skill_0 -i skill_1 -i skill_2",
+    multiple=True,
 )
-def run_handler_check(packages_dir: Path, skip: str, common: str) -> None:
+def run_handler_check(
+    ctx: Context, ignore: List[str], common_handlers: List[str]
+) -> None:
     """Check handler definitions."""
 
-    # TODO: use more appropriate input type and apply validation on it
-    skip_skills = skip.split(",")
-    common_handlers = common.split(",")
-    packages_dir = Path(packages_dir)
-
     try:
-        for yaml_file in sorted(packages_dir.glob(f"**/{DEFAULT_SKILL_CONFIG_FILE}")):
+        for yaml_file in sorted(
+            Path(ctx.registry_path).glob(f"*/*/*/{DEFAULT_SKILL_CONFIG_FILE}")
+        ):
+            if yaml_file.parent.name in ignore:
+                click.echo(f"Skipping {yaml_file.parent}")
+                continue
+
             click.echo(f"Checking {yaml_file.parent}")
-            check_handlers(yaml_file.resolve(), common_handlers, skip_skills)
-    except Exception as e:  # pylint: disable=broad-except
+            check_handlers(
+                yaml_file.resolve(),
+                common_handlers=common_handlers,
+            )
+    except (FileNotFoundError, ValueError, ImportError) as e:
```

Comment:
you reckon we should be using [`.resolve` instead of `.absolute`](https://github.com/python/cpython/blob/0e15c31c7e9907fdbe38a3f419b669fed5bb3b33/Lib/pathlib.py#L915-L950) as a default?